keyboards/cybergear/macro25/config.h

```diff
@@ -0,0 +1,26 @@
+	/* Copyright 2021 CyberGear Marius Kavoliunas
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define MANUFACTURER    CyberGear
+#define PRODUCT         Macro25 (mk-macro-01)
+
+#define MATRIX_ROWS 2
+#define MATRIX_COLS 5
```

Comment:
Does it still compile properly without these? I'm unsure whether they can be computed from the number of pins specified in the info.json, or whether they must be added as `matrix_size`.

Reply:
@fauxparYou were right, those two are not necessary.